R/sparkR-mosaic/sparkrMosaic/DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: sparkrMosaic
 Title: SparkR bindings for Databricks Mosaic
-Version: 0.2.2-SNAPSHOT
+Version: 0.2.2
```

Comment:
if we cant use -SNAPSHOT, for consistency I would suggest adding .9000 (or similar for indicating this is a snapshot version) and document this in the contribute md

Reply:
Requirements for Version field in DESCRIPTION:
The mandatory ‘Version’ field gives the version of the package. This is a sequence of at least two (and usually three) non-negative integers separated by single ‘.’ or ‘-’ characters. The canonical form is as shown in the example, and a version such as ‘0.01’ or ‘0.01.0’ will be handled as if it were ‘0.1-0’. It is not a decimal number, so for example 0.9 < 0.75 since 9 < 75.